bin/gen-pgbouncer-conf.sh

```diff
@@ -32,6 +32,7 @@ ciphers = HIGH:!ADH:!AECDH:!LOW:!EXP:!MD5:!3DES:!SRP:!PSK:@STRENGTH
 debug = ${PGBOUNCER_STUNNEL_LOGLEVEL:-notice}
 EOFEOF
 
+mkdir -p /app/vendor/pgbouncer
```

Comment:
When writing `test/gen-pgbouncer-conf.sh.bats`, I found this bug in the original project!

In the wild, everything works because `bin/compile` makes this directory.

But considering `bin/gen-pgbouncer-conf.sh` in isolation, it fails.  And is inconsistent.  Earlier in this file we make `/app/vendor/stunnel/` before we make `/app/vendor/stunnel/stunnel.conf`.